packages/react/index.js

```diff
@@ -1,6 +1,38 @@
 'use strict';
 
+function testMinification() {
+  if (testMinification.name === 'testMinification') {
+    // We are not minified.
+    // This might be a Node environment where DCE is not expected anyway.
+    return;
+  }
+  if (process.env.NODE_ENV === 'development') {
+    // We expect this method only to be called in production.
+    throw new Error('This is unreachable');
+  }
+  try {
+    const source = testMinification.toString();
+    if (source.indexOf('toString') === -1) {
+      // We know for a fact the above line exists.
+      // Therefore the browser gave us invalid source.
+      return;
+    }
+    if (source.indexOf('unreachable') !== -1) {
```

Comment:
! Good catch! I'll update it. Maybe we can have it look for two occurrences of the search string.